src/service/force.js

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+'use strict';
+
+const {Record, Any} = require('../common/typed');
```

Comment:
> Should this action live in service directory?

That's a good question, initially it was supposed be a service that deals with this hack https://github.com/mozilla/browser.html/pull/569/files#diff-d0e0be46849a6d8f84059218d61e3fc5R30 but than it ended up inlined for a technical reasons, which is unfortunate but never the less a case. Maybe at this point it indeed would be more appropriate to move it to `common` instead.